src/Servers/IIS/IIS/src/BadHttpRequestException.cs

```diff
@@ -8,6 +8,7 @@
 
 namespace Microsoft.AspNetCore.Server.IIS
 {
+#pragma warning disable CS1591 // Missing XML comment for publicly visible type or member
```

Comment:
Why not use `/// <inheritdoc />` here?

Reply:
The type is deprecated anyways, does it matter that much?

Reply:
Cleanliness, I'd much rather have an inheritdoc than a pragma.